lib/presentation/widgets/login_button_widget.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+
+class CustomButton extends StatelessWidget {
+  const CustomButton({Key? key, required this.onPressed, required this.text})
+      : super(key: key);
+
+  final VoidCallback onPressed;
+  final String text;
+
+  @override
+  Widget build(BuildContext context) {
+    return ElevatedButton(
+      onPressed: onPressed,
+      style: ElevatedButton.styleFrom(
+        minimumSize: const Size.fromHeight(12),
+        padding: const EdgeInsets.all(8.0),
+        primary: const Color(0xFF292D32),
+        shape: RoundedRectangleBorder(
+          //todo set accurate radius
+          borderRadius: BorderRadius.circular(40),
+        ),
+      ),
+      child: Padding(
+        padding: const EdgeInsets.symmetric(vertical: 10.0),
```

Comment:
pls make sizing divided by 4, so 9 or 12 in this case

Reply:
9 not divide by 4